front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -76,7 +86,9 @@ interface InputProps {
   className?: string;
   opened: boolean;
   close: () => void;
-  topBar?: JSX.Element | null;
+  modalTitle?: JSX.Element | null;
+  titleAs?: TitleProps['as'];
+  titleVariant?: TitleProps['variant'];
```

Comment:
Premature optimization but I didn't want to give the impression to the next dev that these can't be changed.